tests/specs/scoped.js

```diff
@@ -277,6 +277,37 @@ test('scoped Store .update()', function (t) {
   })
 })
 
+test('scoped Store .update() updates the correct store', function (t) {
+  t.plan(4)
+
+  var store = new Store('test-db-scoped-update', merge({remote: 'test-db-scoped-update'}, options))
+
+  var testItem1 = { foo: 'bar1', type: 'test1' }
+  var testItem2 = { foo: 'bar2', type: 'test2' }
+  var item1
+
+  store.add([testItem1, testItem2])
+
+  .then(function (items) {
+    item1 = items[0]
+    return store('test2').update([{ id: items[1].id, foo: 'bar20' }, { id: items[0].id, foo: 'bar10' }])
+  })
+
+  .then(function (newItem) {
+    t.is(newItem.length, 1, 'returns only testStore2 item')
```

Comment:
Here I would not agree, if I pass in two objects to an `.update` (no matter if scoped or not), I expect an array of the same length to be returned. If one object cannot be found, an error should be return at its index.

That’s how the `.find` method works, too